module/Database/src/Model/Member.php

```diff
@@ -267,7 +270,7 @@ public function __construct()
     /**
      * Get the membership number.
      *
-     * @return int|null
+     * @psalm-ignore-nullable-return
```

Comment:
At the source, in practice that function never returns `null` because Doctrine takes care of it. We already do the same in GEWISWEB ([see here](https://github.com/GEWIS/gewisweb/blob/74a7d9769119c82c2c3375f11b8e0391faa0fb34/module/Application/src/Model/Traits/IdentifiableTrait.php#L27-L35)).

It fixes the Psalm errors for the `to(Api)Array` `@return` definitions.

Reply:
I'll close the other ones are 'resolved' and we can discuss here if needed.

Reply:
Seems fine